src/ui-kit/components/modal/modal.template.html

```diff
@@ -13,7 +13,7 @@ <h3 id="dialog-title" class="usa-alert-heading">{{title}}</h3>
 			</div>
 		</div>
 		<div *ngIf="this.type === 'primary'" class="primary-model-body">
-			<button #closeButton class="sam-primary-button pull-right" *ngIf="showClose" (click)="closeModal()"><span
+			<button #closeButton aria-label="close" class="sam-primary-button pull-right" *ngIf="showClose" (click)="closeModal()"><span
 					class="fa fa-times-circle" [attr.id]="modalElIds?.closeId"></span><span class="usa-sr-only">close
```

Comment:
So within the button there is a span that contains the text "close modal". Is that not being read? What browser are you testing this change with?